### ISSUE TITLE
Add translations for dark mode toggle button tooltip

### ISSUE THREAD

**Issue by pbaity** [pbaity/rocketchat-dark-mode#33]
The tooltip for the dark mode toggle button is only in English and Russian. In any other language setting it stays in English.

It'd be great for users who are native speakers of the other languages offered in RocketChat to add translations to the tooltip to [dark-mode-toggle.js](https://github.com/pbaity/rocketchat-dark-mode/blob/master/dark-mode-toggle.js#L18). We need translations for:

- [ ] German
- [ ] French
- [ ] Spanish
- [ ] Italian
- [ ] Dutch
- [ ] Polish
- [ ] Portuguese 

P.S. Google Translate is not what we're looking for. "Toggle Dark Mode" isn't communicated the same in other languages. It'd be best for it the tooltip text to read naturally to native speakers, rather than a possibly robotic-sounding translation by Google.

**Comment by rapliu** [comment]
Can I take part in Chinese translation? Although mine is not very good, because I know rocket very well, I know what Chinese users need

**Comment by pbaity** [comment]
@rapliu Please do! The languages I listed above are the ones I found codes for in Rocket.Chat, but I'm sure that's not an exhaustive list. In [dark-mode-toggle.js](), in the `const darkModeToggleText` object, just add a line for Chinese. 

Don't worry if it's not a perfect translation - if another Chinese-speaking user notices a problem with it, we can fix it then.

**Comment by pbaity** [comment]
@y377 Still interested in offering a Chinese translation?

**Comment by fdellwing** [comment]
In German `Dark Mode umschalten` should be fine. We have not really a German term for dark mode.

**Comment by pbaity** [comment]
Thanks @fdellwing , added in #45

**Comment by aradhya-gupta** [comment]
Although Hindi is not mentioned above, shall I add a Hindi translation for the tool tip?

**Comment by pbaity** [comment]
@aradhya-gupta Please do! I took the list above from [this file](https://github.com/RocketChat/Rocket.Chat/blob/0750cb6af2643fe055f31d684d13bf9b21fc799c/app/autotranslate/server/deeplTranslate.js#L74) in the RocketChat repository, but I think these are just the languages that support auto-translation. I'm adding Hindi to the list now.

**Comment by bseclier** [comment]
Franch traduction is : "Activer le mode sombre".

**Comment by pbaity** [comment]
Thanks @bseclier , after a long holiday break I added your translation in #102.

**Comment by y377** [comment]
Yeah, I've been waiting for you to assign me something to translate
